polaris.shopify.com/content/patterns/app-settings-layout/variants/default.md

```diff
@@ -46,19 +46,19 @@ This pattern uses the [`AlphaStack`](/components/layout-and-structure/alpha-stac
   ]}
 >
   <AlphaStack gap="16" align="center">
-    <Columns columns={{xs: '1fr', md: '2fr 5fr'}}>
+    <Columns columns={{xs: '1fr', md: '2fr 5fr'}} gap={{xs: '4', md: '0'}}>
```

Comment:
```suggestion
    <Columns columns={{xs: '1fr', md: '2fr 5fr'}} gap="4">
```

When stacked vertically, we still want a gap of `"4"`

Reply:
That's how this reads, we want it when stacked vertically, but not when its a 2-column layout